.github/workflows/karma.yml

```diff
@@ -95,9 +95,45 @@ jobs:
 
       - run: API_VERSION=60 yarn sauce:ci
       - run: API_VERSION=60 DISABLE_SYNTHETIC=1 yarn sauce:ci
+      - run: API_VERSION=61 yarn sauce:ci
+      - run: API_VERSION=61 DISABLE_SYNTHETIC=1 yarn sauce:ci
       - run: ENABLE_ARIA_REFLECTION_GLOBAL_POLYFILL=1 yarn sauce:ci
       - run: ENABLE_ARIA_REFLECTION_GLOBAL_POLYFILL=1 DISABLE_SYNTHETIC=1 yarn sauce:ci
       - run: DISABLE_SYNTHETIC_SHADOW_SUPPORT_IN_COMPILER=1 DISABLE_SYNTHETIC=1 yarn sauce:ci
+
+      - name: Upload coverage results
+        uses: actions/upload-artifact@v3
+        with:
+          name: coverage-report-group-2
+          path: ./packages/@lwc/integration-karma/coverage
+
+  run-karma-tests-group-3:
+    runs-on: ubuntu-22.04
+    env:
+      SAUCE_TUNNEL_ID: github-action-tunnel-integration-${{github.run_id}}-group-3
+    defaults:
+      run:
+        working-directory: ./packages/@lwc/integration-karma
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: "18.18.0"
+          cache: 'yarn'
```

Comment:
Is having `"18.18.0"` three times enough to extract it to a variable for easier future updates? Also, mixing `"` and `'` 🥳

Reply:
I looked into this previously, and there is no easy way to [share code](https://stackoverflow.com/a/71570780) in GitHub Actions without creating a separate repo. I can fix the mixed quotes though.

Reply:
Ah maybe I can extract out just the variable though? Let me try.

Reply:
Done!